pantry/pantry_method_script.sh

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
-
-set -o errexit
-set -o pipefail
-set -o xtrace
-
-. /lib/svc/share/smf_include.sh
-
-DATALINK="$(svcprop -c -p config/datalink "${SMF_FMRI}")"
-GATEWAY="$(svcprop -c -p config/gateway "${SMF_FMRI}")"
-LISTEN_ADDR="$(svcprop -c -p config/listen_addr "${SMF_FMRI}")"
-LISTEN_PORT="$(svcprop -c -p config/listen_port "${SMF_FMRI}")"
-
-if [[ $DATALINK == unknown ]] || [[ $GATEWAY == unknown ]]; then
-    printf 'ERROR: missing datalink or gateway' >&2
-    exit "$SMF_EXIT_ERR_CONFIG"
-fi
-
-# TODO remove when https://github.com/oxidecomputer/stlouis/issues/435 is addressed
-ipadm delete-if "$DATALINK" || true
```

Comment:
Same question here, setting the MTU was important at one time, so before
we remove this, I would want to be sure it's done elsewhere.

Reply:
same as above